windows-server-container-tools/New-ContainerHost/New-ContainerHost.ps1

```diff
@@ -242,7 +242,7 @@ Cache-HostFiles
 
     if ($global:DeveloperMode -and ($global:localVhdName -match "\.th2_release\."))
     {
-        Copy-File -SourcePath "http://aka.ms/containerzdp" -DestinationPath "$global:localVhdRoot\zdp.cab"
+        Copy-File -SourcePath "https://aka.ms/containerzdp" -DestinationPath "$global:localVhdRoot\zdp.cab"
```

Comment:
This URL is redirected back to an http URL http://10.137.119.43/Windows10.0-KB3081439-x64.cab